app/boards/shields/swept/boards/nice_nano.overlay

```diff
@@ -0,0 +1,33 @@
+#include <dt-bindings/led/led.h>
```

Comment:
I'd also consider adding an overlay for other common boards, at least for `nice_nano_v2` until a better solution to #885 is implemented.

Reply:
Actually, this board only supports v2, so I'll change it. Thanks